lib/cli/github_settings/do_check.js

```diff
@@ -123,175 +135,200 @@ function do_check(subcmd, opts, args, cb) {
-                                });
-                            }
-                            if (prot.restrictions) {
-                                checkFailures.push({
-                                    repo: repo.name,
-                                    name: 'branch_protection.restrictions',
-                                    msg:
-                                        '"Restrict who can push to matching branches" must not be checked'
-                                });
-                            }
-
-                            next();
-                        })
-                        .catch(function(err) {
-                            if (err.message === 'Branch not protected') {
-                                checkFailures.push({
-                                    repo: repo.name,
-                                    name: 'branch_protection',
-                                    msg: 'Must have "master" branch protection'
-                                });
-                                next();
-                            } else if (err.message === 'Branch not found') {
-                                checkWarnings.push({
-                                    repo: repo.name,
-                                    name: 'branch_protection',
-                                    msg: 'No "master" branch (skipping)'
-                                });
-                                next();
-                            } else {
-                                // Avoid deprecation warning by wrapping the error but
-                                // still allowing cmdln to use `err.code`.
-                                //   Deprecation: [@octokit/request-error] `error.code` is deprecated, use `error.status`.
-                                next(
-                                    new VError(
-                                        {
-                                            name: 'GitHubApiError',
-                                            cause: err,
-                                            code: err.status,
-                                            info: {
-                                                repo: 'joyent/' + repo.name
-                                            }
-                                        },
-                                        'error calling GitHub API'
-                                    )
-                                );
-                            }
-                        });
+                function checkMasterBranchProtection(ctx, next) {
+                    checkBranchProtection('master', ctx, next);
+                },
+                function checkMantaV1BranchProtection(ctx, next) {
+                    checkBranchProtection('mantav1', ctx, next);
                 }
-
-                // TODO: add merge button check (get repo api)
-                // TODO: add wiki check (get repo api)
             ]
```

Comment:
Nit: These TODO comments could stay here in this area. They wouldn't be part of the "checkBranchProtection" function if we ever get to them.

Reply:
Of course, yep.